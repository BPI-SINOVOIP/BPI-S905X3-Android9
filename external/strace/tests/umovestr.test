#!/bin/sh

# umovestr short read regression test

. "${srcdir=.}/init.sh"

run_prog
run_strace -e chdir $args
match_diff

exit 0
