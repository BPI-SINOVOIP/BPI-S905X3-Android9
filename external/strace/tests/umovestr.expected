chdir("////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////") = 0
+++ exited with 0 +++
